packages/governance/src/contractGovernor.js

```diff
@@ -28,7 +28,6 @@ const validateQuestionDetails = async (zoe, electorate, details) => {
   } = details;
   fit(details, ParamChangesQuestionDetailsShape);
 
-  /** @type {import('@agoric/zoe/src/zoeService/utils.js').Instance<typeof start>} */
```

Comment:
`getTerms` now infers from the start function parameter of its argument (here `governedInstance`)